.github/workflows/distribute.yml

```diff
@@ -0,0 +1,22 @@
+name: Distribute
+
+on:
+  workflow_run:
+    workflows: [ Build ]
+    types: [ completed ]
+    branches: [ dev ]
+
+jobs:
+  distribute-android:
+    runs-on: ubuntu-latest
+    container: gradle:8.5.0-jdk17
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/gradle-cache
+      - uses: android-actions/setup-android@v3.2.0
+      - uses: ./.github/actions/firebase-google-service
+        with:
+          secret: ${{ secrets.FIREBASE_GOOGLE_SERVICE }}
+      - run: ./gradlew assembleDebug
```

Comment:
Duplicated assemble task will be removed and migrated to using outputs from "Build" workflow. Unfortunately we need to merge it to the default branch first due to limitation of `workflow_run` event _This event will only trigger a workflow run if the workflow file is on the default branch._